typed/parser_test.go

```diff
@@ -86,8 +86,10 @@ func BenchmarkFromUnstructured(b *testing.B) {
 			b.ReportAllocs()
 			b.ResetTimer()
 			for n := 0; n < b.N; n++ {
-				if _, err := pt.FromUnstructured(obj); err != nil {
+				if u, err := pt.FromUnstructured(obj); err != nil {
 					b.Fatal(err)
+				} else {
+					_ = u.AsValue().ToUnstructured(false)
```

Comment:
Why not add a second b.Run(...) instead of making the existing one less clear?

Also, should check the error and b.Fatal if non-nil?

Reply:
`ToUnstructure` can't fail. Yeah, I'll make another test ...

Reply:
Updated.